physics/satmedmfvdifq.F

```diff
@@ -89,6 +95,7 @@ subroutine satmedmfvdifq_run(im,km,progsigma,ntrac,ntcw,ntrw,     &
       integer, intent(in)  :: im, km, ntrac, ntcw, ntrw, ntiw,          &
      &                        ntke, ntqv
       integer, intent(in)  :: sfc_rlm
+      integer, intent(in)  :: tc_pbl
```

Comment:
If this is only being checked for 0/1, why not use a logical here?

Reply:
@grantfirl, tc_pbl in the future could have options like 2, 3, etc. Please see in the response to your other comments as well. Thanks!

Reply:
OK, sounds good. In that case, we've been trying to use CCPP standard names of `control_for_...`. Could you change the standard name of this variable to `control_for_hurricane_PBL_modifications` or something similar?

Reply:
@grantfirl, Thanks for the suggestion! Do the following standard and long names sound good to you?
```
[tc_pbl]
  standard_name = control_for_tc_pbl_modifications_in_boundary_layer_mass_flux_scheme
  long_name = control for tc pbl modifications in boundary layer mass flux scheme
```
If so, I will make the changes accordingly and then commit/push the changes in. Thanks!

Reply:
Since the tc_pbl option leads to a severe reduction in the GFS medium-range forecast skill, I recommend to apply it only to the HAFS if one would like to.

Reply:
We would definitely like to have the option available for current and future HAFS applications. Ongoing testing shows notable improvement using this option in HAFS-B.

Reply:
@ChunxiZhang-NOAA, Thanks for the suggestion! Will change accordingly.

Reply:
> Maybe you can switch to this standard_name: control_for_TC_applications_in_the_PBL_scheme

@ChunxiZhang-NOAA and @grantfirl, tc_pbl's standard and long names have been updated accordingly in the latest commit. Thanks!

Reply:
@BinLiu-NOAA Thanks. Looks good.